types/core/speedy-media-source.d.ts

```diff
@@ -73,5 +73,5 @@ export class SpeedyMediaSource {
      */
     _load(element: SpeedyMediaSourceNativeElement): SpeedyPromise<SpeedyMediaSource>;
 }
-export type SpeedyMediaSourceNativeElement = HTMLImageElement | HTMLVideoElement | HTMLCanvasElement | ImageBitmap;
+export type SpeedyMediaSourceNativeElement = HTMLImageElement | HTMLVideoElement | HTMLCanvasElement | any | ImageBitmap | ImageData;
```

Comment:
Tsc export OffscreenCanvas as any instead ImageData is correct. Maybe OffscreenCanvas should be added to MediaType?